docs/changelog_3_4_0.rst

```diff
@@ -1,6 +1,210 @@
+|
+| **Read before updating**:
+| 1. This Red update bumps discord.py to version 1.5.1, which explicitly requests Discord intents. Red requires all Prvileged Intents to be enabled. More information can be found at :ref:`enabling-privileged-intents`.
+| 2. Mutes functionality has been moved from Mod cog to a new separate cog (Mutes) featuring timed and role-based mutes. If you were using it (or want to start now), you can load the new cog with ``[p]load mutes``. You can see full `Mutes changelog below <important-341-1>`.
+| 3. Information for Audio users that are using external Lavalink instance (if you don't know what that is, you should skip this point):
+| We've updated our ``application.yml`` file and you should update your instance's ``application.yml`` appropriately.
+  Please ensure that the WS port in Audio's settings (``[p]llset wsport``) is set to the port from the ``application.yml``.
+
+End-user changelog
+------------------
+
+Core Bot
+********
+
+- Fixed incorrect error being reported on ``[p]set name`` when the passed name was longer than 32 characters (:issue:`4364`, :issue:`4363`)
+- Fixed ``[p]set nickname`` erroring when the passed name was longer than 32 characters (:issue:`4364`, :issue:`4363`)
+- Fixed an ungraceful error being raised when running ``[p]traceback`` with closed DMs (:issue:`4329`)
+- Fixed errors that could arise from invalid URLs in ``[p]set avatar`` (:issue:`4437`)
+- Fixed error being raised with ``[p]set nickname`` when no nickname was provided (:issue:`4451`)
+- Fixed and clarified errors being raised with ``[p]set username`` (:issue:`4463`)
+- Fixed ungraceful error raising when ``[p]unload`` output is larger than 2k characters (:issue:`4469`)
+- Fixed ungraceful error raising when providing ``[p]choose`` with empty options (:issue:`4499`)
+- Fixed ungraceful error raising when a bot left a guild while a menu was open (:issue:`3902`)
+- Fixed info missing on non-embed version of ``[p]debuginfo`` (:issue:`4524`)
+- Added ``[p]set api list`` to list all currently set API services, without tokens (:issue:`4370`)
+- Added ``[p]set api remove`` to remove API services, including tokens (:issue:`4370`)
+- Added ``[p]helpset usetick``, toggling command message being ticked when help is sent to DM (:issue:`4467`, :issue:`4075`)
+- Added default color field to ``[p]set showsettings`` (:issue:`4498`, :issue:`4497`)
+- Added datapath and metadata file to ``[p]debuginfo`` (:issue:`4524`)
+- Added list of disabled intents to ``[p]debuginfo`` (:issue:`4423`)
+- Bumped discord.py dependency to version 1.5.1 (:issue:`4423`)
+- Locales and regional formats can now be set in individual guilds using ``[p]set locale`` and ``[p]set regionalformat`` (:issue:`3896`, :issue:`1970`)
+
+    - Global locale and regional format setters have been renamed to ``[p]set globallocale`` and ``[p]set globalregionalformat``
+
+Audio
+*****
+
+- Fixed scattered grammar and typo fixes (:issue:`4446`)
```

Comment:
```suggestion
- Scattered grammar and typo fixes (:issue:`4446`)
```

Reply:
That changes the meaning. This fixed the grammar being scattered. It did not scatter the grammer.